app/javascript/client/form/NumberInput.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { HTMLProps } from 'react';
+import styled from 'styled-components';
+
+interface NumberInputProps extends Omit<HTMLProps<HTMLInputElement>, 'as' | 'ref' | 'autoComplete'> {
+}
+
+const StyledNumberInput = styled.input`
+`;
+
+export const NumberInput = React.forwardRef(( { ...rest }: NumberInputProps, ref) => (
+    <StyledNumberInput {...rest} ref={ref as React.RefObject<HTMLInputElement>} type="number" min="1" max="999" />
```

Comment:
Can do this in a future PR, but Jim says he will occasionally get an order for 1,000+, so let's make the max 3,000